### DISCUSSION TITLE
Long-term contributor to Uiua page on APL wiki?

### DISCUSSION THREAD

**Discussion by ghost** [uiua-lang/uiua#515] [General]
At 27 September 2023, the page for Uiua in the APL Wiki is create. It have way less information, just the overview and infobox. Then, 4 month later, I edit the page to include tables of almost all the primitive with glyph at the time (0.8.0). Then, six month later and I edit it ones more. And I look at the page. It has not change too much! Some primitive was remove, due to them removed in the Uiua language, a little bit of reformatting, but that it. I do not see any new primitive, examples, anything!(For the 6 month gap, it because I bored and quit, but the page still in my mind. When I feel good, I edit it.)
So, do you planed to hire a person to update the page up to date?

**Comment by kaikalii** [comment]
No, I do not plan to hire anyone.

While keeping the APL wiki up to date my be nice for some, I'm not really concerned with it.

**Comment by JoaoFelipe3** [comment]
I updated the page a few days ago (5th of July), mainly adding some primitives and removing some unnecessarily long descriptions.
